### PR TITLE
LPS-160632 Add 'custom'/'system' labels when we are choosing an object in Modal to add a relationship

### PR DESCRIPTION
**Pull request by danvb112**
Related Issues: 

https://issues.liferay.com/browse/LPS-160632
https://issues.liferay.com/browse/LPS-160389

### CODE REVIEW

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/ObjectRelationshipFormBase.tsx [hunk 1]
```diff
@@ -165,6 +170,23 @@ export function ObjectRelationshipFormBase({
...
+			});
+		}
+		else {
+			return filteredObjectDefinitions.filter(({label}) => {
+				return label[defaultLanguageId]
+					?.toLocaleLowerCase()
+					.includes(query.toLowerCase());
+			});
+		}
+	}, [objectDefinitions, filteredObjectDefinitions, query]);
```
Comment (rebsilva) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/ObjectRelationshipFormBase.tsx:188]: I think these should be in alphabetical order, like `[filteredObjectDefinitions, objectDefinitions, query]`

**Review by rebsilva — CHANGES_REQUESTED** [review]
Hi @danvb112! Great job on this PR, I just requested a fix for a minor formatting issue :cherry_blossom: Thanks!

**Comment by rebsilva** [comment]
Also @danvb112, taking a quick look into the failing tests I think they are related to your changes, since the element is no more a select but a autocomplete instead. One of the errors says this:

`Element is not present at "//select[@class = 'form-control']"`

Can you please take a look and confirm this? Thanks!

**Comment by danvb112** [comment]
ci:test:sf

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/ObjectRelationshipFormBase.tsx (hunk 1 above)
Comment (danvb112) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/ObjectRelationshipFormBase.tsx]: Done! Thanks!

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:objects

**Comment by danvb112** [comment]
ci:test:object

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
ci:test:relevant

**Review by rebsilva — APPROVED** [review]
LGTM! :cherry_blossom:

**Review by anabuchmann — APPROVED** [review]
Hi @danvb112 !! 
The Poshi code changes look good to me. Good job! Thanks.

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
ci:forward

**Comment by danvb112** [comment]
ci:forward

**Comment by rebsilva** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
ci:forward

**Pull request closed without merging**